cumulus_etl/fhir/fhir_utils.py

```diff
@@ -190,6 +218,7 @@ async def get_docref_note(client: fhir.FhirClient, docref: dict) -> str:
 
     # Strip this "line feed" character that often shows up in notes and is confusing for NLP.
     # Hopefully not many notes are using actual Spanish.
-    note = note and note.replace("¿", " ")
+    note = note.replace("¿", " ")
```

Comment:
Unrelated, but a while back I changed this method to not use `None` notes and instead just raise exceptions on errors. So this `note and` check is unnecessary now and gives the wrong impression that `note` could be `None`.